coverage/.resultset.json

```diff
@@ -1,26 +1,26 @@
 {
```

Comment:
You'll want to remove the coverage folder from the git cache:

"call `git rm --cached` on each of the files you want to remove from version control"

And then commit removing the files from source control!